.travis.yml

```diff
@@ -15,9 +15,15 @@ addons:
       # Speedup builds, particularly when USE_CHROOT=1
       - eatmydata
 
+branches:
+  only:
+    - master      
```

Comment:
I don't think this is right, releases are done from bugfix branches, and travis should be passing on those, too